app/renderer/components/main/braveryPanel.js

```diff
@@ -184,164 +240,692 @@ class BraveryPanel extends ImmutableComponent {
+                })}
+                  customInfoButton={css(styles.braveryPanel__body__advanced__control__switchControl__infoButton)}
+                  onClick={this.onToggleFp}
+                  rightl10nId='fingerprintingProtection'
+                  checkedOn={fpEnabled}
+                  disabled={!shieldsUp}
+                  onInfoClick={this.onInfoClick}
+                  infoTitle={config.fingerprintingInfoUrl}
+                  testId='fingerprintingProtectionSwitch'
+                />
+                <SwitchControl className={cx({
+                  [css(gridStyles.row4col2)]: !compactBraveryPanel,
+                  [css(gridStyles.row8col1)]: compactBraveryPanel,
+                  [css(styles.braveryPanel_compact__body__advanced__control__switchControl)]: compactBraveryPanel
+                })}
+                  onClick={this.onToggleSafeBrowsing}
+                  rightl10nId='safeBrowsing'
+                  checkedOn={this.props.braverySettings.safeBrowsing}
+                  disabled={!shieldsUp}
+                  testId='safeBrowsingSwitch'
+                />
+              </div>
+            </section>
             : null
           }
-          <hr className='braveryBottomSplitter' />
-          <div className='braveryPanelFooter'>
-            <span className='clickable' onClick={this.onEditGlobal} data-l10n-id='editBraveryGlobalSettings' />
-            <span className='reloadButton clickable' onClick={this.onReload}>
-              <div className='reloadText' data-l10n-id='reload' />
-              <div className='fa fa-repeat' />
-            </span>
+          <hr className={css(
+            styles.braveryPanel__body__hr,
+            styles.braveryPanel__body__hr_splitter,
+            compactBraveryPanel && styles.braveryPanel_compact__body__hr
+          )} />
+          <div className={css(styles.braveryPanel__body__footer)}>
+            <span className={css(
+              styles.braveryPanel__body__footer__edit,
+              styles.braveryPanel__body__footer__edit_clickable,
+              compactBraveryPanel && styles.braveryPanel_compact__body__footer__edit
+            )}
+              onClick={this.onEditGlobal}
+              data-l10n-id='editBraveryGlobalSettings'
+            />
+            <div className={css(
+              styles.braveryPanel__body__footer__reload,
+              styles.braveryPanel__body__footer__reload_clickable
+            )}
+              onClick={this.onReload}>
+              <span className={css(styles.braveryPanel__body__footer__reload__text)} data-l10n-id='reload' />
+              <span className='fa fa-repeat' />
+            </div>
           </div>
-        </div>
+        </section>
       </div>
     </Dialog>
   }
 }
 
+const displayHost = {
+  fontSize: '20px',
+  fontWeight: 'normal',
+  textOverflow: 'ellipsis',
+  whiteSpace: 'nowrap',
+  overflow: 'hidden'
+}
+const editGlobalMarginBottom = '.25rem'
```

Comment:
Since I set `1rem` to padding of `braveryPanel_compact__body` and I wanted to set `.25rem` as 25% of that padding. We know 1rem=16px, so it should be clear that `.25rem=16px*25%` and `.75rem=16px*75%` without doing math (no matter how easy it is in this case).

Using `px` sometimes makes it hard to know how and why this value has been set so. that's why I prefer relative units for elements which I added by myself. I avoided converting the units of other elements which already have existed because we can trust the value.